src/utils/findDirectoryNode.ts

```diff
@@ -1,6 +1,9 @@
 import directory from '@/directory/directory.json';
+import { PageNode } from '@/directory/directory';
 
-export const findDirectoryNode = (route, dir = directory) => {
+const directoryCast = directory as PageNode;
```

Comment:
Doing this so there isn't a typescript warning when we recurse on `findDirecotryNode` on line 12. This is because the root directory node (which is the home/index page) doesn't have a `platforms` property